### DISCUSSION TITLE
How to create custom section of links?

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#969] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45195351/177742649-fd0081c2-e282-4f01-83d2-2c4496df20c4.png)

Whenever I generate an md file to show links it adds an "S" to the title which is weird.

**Comment by adityatelange** [comment, marked as answer]
Duplicate of https://github.com/adityatelange/hugo-PaperMod/discussions/338
